x/concentrated-liquidity/fees_test.go

```diff
@@ -1015,3 +1023,168 @@ func (s *KeeperTestSuite) TestPreparePositionAccumulator() {
 		})
 	}
 }
+
+type Positions struct {
+	numAccounts    int
+	numFullRange   int
+	numNarrowRange int
+	numConsecutive int
+	numOverlapping int
+}
+
+func (s *KeeperTestSuite) TestFunctionalFees() {
+	positions := Positions{
```

Comment:
nit: could do the same fo numSwaps

Reply:
https://github.com/osmosis-labs/osmosis/pull/4282/commits/e5e3810b15f2f19ad29af40dea28c931c62a3648